mftf/2.3/debugging.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Debugging MFTF Tests
+mftf-release: 2.3.13
+---
+
+Debugging within the Magento Functional Testing Framework is helpful in identifying test bugs by allowing you to pause execution so that you may:
+
+- Examine the page.
+- Check returned data and other variables being used during run-time.
+
+This is straightforward to do once you create a basic Debug Configuration.
+
+## Prerequisites
+
+- [Xdebug]
+- PHPUnit configured for use in [PHPStorm]
+
+## Creating Debug Configuration with PHPStorm
+
+1. Download the Codeception Framework plugin for PHPStorm (`PhpStorm->Preferences->Plugins`).
```

Comment:
It may be worth noting that the Codeception plugin is probably already installed. It is bundled with PHPStorm.

Reply:
Noted.